README.md

```diff
@@ -85,3 +85,11 @@ interface.
 GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
 
 To learn how to report a security vulnerability, [see our security policy](https://github.com/alphagov/govuk-design-system/security/policy).
+
+## Contributing
```

Comment:
Do we need to link to the [Frontend contribution guidelines](https://github.com/alphagov/govuk-frontend/blob/f07569c9e7724606970c0d2764f1e9ce7fa14092/CONTRIBUTING.md) somewhere beneath here? Or can we skip that because the link is to a different repo?

Reply:
I think we can skip that here.